src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/shops.yml

```diff
@@ -3,5 +3,6 @@ admin_shops_search:
   methods: [ GET ]
   defaults:
     _controller: 'PrestaShopBundle\Controller\Admin\Configure\AdvancedParameters\ShopController::searchAction'
+    _legacy_controller: AdminAdvancedParameters
```

Comment:
As @matks said it seems this controller is used across the whole BO for the multi shop headers, it may need to be open to all users And is definitely not related to permissions about advanced parameters